CHANGES.md

```diff
@@ -6,6 +6,13 @@ This release supports [version
 
 ## Changes
 
+* The `LLVMSetup` and `MIRSetup` monad types are now just ordinary builtin
+  types; their names are no longer reserved words.
+  (The `JVMSetup` type was already this way.
+  A bogus unused `JavaSetup` reserved word has been removed.
+  See #2768.)
+  The legacy name `CrucibleSetup` remains a reserved word for the time being.
```

Comment:
It might be worth opening a separate issue about eventually deprecating `CrucibleSetup` in favor of `LLVMSetup`. Some other names that might be worth deprecating include `CrucibleMethodSpec` (in favor of `LLVMSpec`) and `SetupValue` (in favor of `LLVMValue`).